src/js/components/RangeSelector/EdgeControl.js

```diff
@@ -48,22 +48,51 @@ class EdgeControl extends Component {
         theme.rangeSelector.edge &&
         theme.rangeSelector.edge.type) ||
       'disc';
+    let node;
+    if (type === 'bar') {
+      node = (
+        <Box
+          flex
+          justifySelf="stretch"
+          width={`${size}px`}
+          background={normalizeColor(color || 'control', theme)}
+          border={
+            focused ? { color: normalizeColor('focus', theme) } : undefined
+          }
+        />
+      );
+    } else if (type === 'disc') {
+      node = (
+        <Box
+          width={`${size + (focused ? 2 : 0)}px`}
+          height={`${size + (focused ? 2 : 0)}px`}
+          round="full"
+          background={normalizeColor(color || 'control', theme)}
+          border={
+            focused ? { color: normalizeColor('focus', theme) } : undefined
+          }
+          style={theme.edge && theme.edge.extend}
```

Comment:
Shouldn't this be `theme.rangeSelector.edge.extend`?

Reply:
this line is a leftover code and I'll remove it. after our last discussion, I thought we don't need the `extend` option since we are providing the overall node to be customized. 
If you feel like we may still want this functionality I can file an issue or submit it on a separate PR. thanks for catching the leftover line.